wormchain/app/app.go

```diff
@@ -725,6 +762,10 @@ func (app *App) RegisterTendermintService(clientCtx client.Context) {
 	tmservice.RegisterTendermintService(app.BaseApp.GRPCQueryRouter(), clientCtx, app.interfaceRegistry)
 }
 
+func (app *App) GetWasmKeeper() *wasmkeeper.Keeper {
+	return &app.wasmKeeper
+}
```

Comment:
Why is this function needed?

Reply:
This is required for the x/tokenfactory/bindings/ test cases to access the wasm keeper to perform store/instantiate/execute/query using test contracts to test the wasm bindings. Other chains have wasmkeeper as a public field that can be accessed by the test cases instead of using a method.